torch/lib/c10d/reducer.cpp

```diff
@@ -22,6 +22,19 @@ inline int64_t current_time_in_nanos() {
   return torch::autograd::profiler::getTime();
 }
 
+// FIXME: Should make this a static method of C++ comm hook, and call it there.
```

Comment:
maybe worth addressing here to avoid code duplication?

Reply:
It was a shortcut for debugging purpose. Will fix it in the next PR. This PR already has quite some noises.